cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java

```diff
@@ -113,16 +114,21 @@ public String getFormat() {
     }
 
     @Override
-    public Network importData(ReadOnlyDataSource ds, NetworkFactory networkFactory, Properties p) {
+    public Network importData(ReadOnlyDataSource ds, NetworkFactory networkFactory, Properties p, Reporter reporter) {
+        Objects.requireNonNull(ds);
+        Objects.requireNonNull(networkFactory);
+        Objects.requireNonNull(reporter);
         TripleStoreOptions options = new TripleStoreOptions();
         String sourceForIidmIds = Parameter.readString(getFormat(), p, SOURCE_FOR_IIDM_ID_PARAMETER, defaultValueConfig);
         if (sourceForIidmIds.equalsIgnoreCase(SOURCE_FOR_IIDM_ID_MRID)) {
             options.setRemoveInitialUnderscoreForIdentifiers(true);
         } else if (sourceForIidmIds.equalsIgnoreCase(SOURCE_FOR_IIDM_ID_RDFID)) {
             options.setRemoveInitialUnderscoreForIdentifiers(false);
         }
+        options.setReporter(reporter.createSubReporter("Read", "Read"));
```

Comment:
Maybe a more comprehensive key? `CGMESTriplestore`?

Reply:
changed the key